.github/workflows/test.yml

```diff
@@ -3,15 +3,7 @@ on:
   push:
     branches:
       - master
-    paths:
-      - '**.java'
-      - '**.gradle'
-      - 'brut.apktool/apktool-lib/src/main/resources/**'
   pull_request:
-    paths:
-      - '**.java'
-      - '**.gradle'
-      - 'brut.apktool/apktool-lib/src/main/resources/**'
```

Comment:
So these were intentional so my my doc updates/readme don't push off a huge test suite. I don't want to remove these. They should be updated to include the config files from actions so they are included.